src/main/java/ch/uzh/ifi/hase/soprafs22/entity/Chat.java

```diff
@@ -46,17 +45,30 @@ public List<Message> getMessages(int fromIndex, int toIndex) {
     // now, fromIndex must be < size, and toIndex too.
     return messages.subList(fromIndex, toIndex);
   }
+    /**
+     * Gets one Message from the chat
+     * @param index: the index of the message
+     * @return Message: the Message on position index
+     */
+    public Message getMessages(int index){
+        if (messages.isEmpty()){
+            return null;
+        }
+        return messages.get(index);
+    }
+
+    /**
+     * Gets all the messages
+     * @return Message: List of all messages
+     */
+    public List<Message> getMessages() {
+        // return all
+        return messages;
+    }
```

Comment:
When do we need this? (Question)

Reply:
i thought maybe the chat has a function to get all written messages?

Reply:
I would rather add a chat.size() function, so that we can use getMessages(0, chat.size()) if (and we won't) need to get all messages ;)